backend/project-management-system/src/main/java/org/egov/works/enrichment/ProjectEnrichment.java

```diff
@@ -0,0 +1,130 @@
+package org.egov.works.enrichment;
+
+import digit.models.coremodels.AuditDetails;
+import digit.models.coremodels.IdResponse;
+import org.apache.commons.lang3.StringUtils;
+import org.egov.common.contract.request.RequestInfo;
+import org.egov.tracer.model.CustomException;
+import org.egov.works.config.ProjectConfiguration;
+import org.egov.works.repository.IdGenRepository;
+import org.egov.works.util.ProjectServiceUtil;
+import org.egov.works.web.models.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Service
+public class ProjectEnrichment {
+
+    @Autowired
+    private ProjectServiceUtil projectServiceUtil;
+
+    @Autowired
+    private IdGenRepository idGenRepository;
+
+    @Autowired
+    private ProjectConfiguration config;
+
+    public void enrichCreateProject(ProjectRequest request) {
```

Comment:
enrichProjectOnCreate

Reply:
Method name changed to enrichProjectOnCreate